integration_tests/integration/submit-select-teams.spec.ts

```diff
@@ -0,0 +1,102 @@
+import AllocateCasesByTeamPage from '../pages/allocate-cases-by-team'
+import Page from '../pages/page'
+import SelectTeamsPage from '../pages/select-teams'
+
+context('Select teams and show allocate cases by team', () => {
+  let allocateCasesByTeamPage
+
+  context('Multiple teams', () => {
+    beforeEach(() => {
+      cy.task('reset')
+      cy.task('stubSignIn')
+      cy.task('stubAuthUser')
+      cy.task('stubGetAllocations')
+      cy.task('stubGetTeamsByPdu')
+      cy.task('stubGetUnallocatedCasesByTeams', {
+        teamCodes: 'TM1,TM2',
+        response: [
+          {
+            teamCode: 'TM1',
+            caseCount: 1,
+          },
+          {
+            teamCode: 'TM2',
+            caseCount: 2,
+          },
+        ],
+      })
+      cy.task('stubWorkloadCases', {
+        teamCodes: 'TM1,TM2',
+        response: [
+          {
+            teamCode: 'TM1',
+            totalCases: 3,
+            workload: 77,
+          },
+          {
+            teamCode: 'TM2',
+            totalCases: 4,
+            workload: 88,
+          },
+        ],
+      })
+      cy.task('stubGetTeamsByCodes', {
+        codes: 'TM1,TM2',
+        response: [
+          {
+            code: 'TM2',
+            name: 'Team 2',
+          },
+          {
+            code: 'TM1',
+            name: 'Team 1',
+          },
+        ],
+      })
+      cy.task('stubPutUserPreferenceTeams')
```

Comment:
Is this verified anywhere? Not sure cypress detects whether stubs have been called